client/lib/importer/actions.js

```diff
@@ -129,14 +129,12 @@ export function finishUpload( importerId, importerStatus ) {
 	} );
 }
 
-export function mapAuthor( importerId, sourceAuthor, targetAuthor ) {
-	Dispatcher.handleViewAction( {
-		type: IMPORTS_AUTHORS_SET_MAPPING,
-		importerId,
-		sourceAuthor,
-		targetAuthor
-	} );
-}
+export const mapAuthor = ( importerId, sourceAuthor, targetAuthor ) => ( {
```

Comment:
@gziolo maybe a test for the reducer when it gets reworked? I'm not sure that a test would add much value here since it basically just passes along its parameters into an object.